tests/integrations/test_keras.py

```diff
@@ -106,9 +103,18 @@ def test_keras_telemetry(
     assert telemetry and 8 in telemetry.get("3", [])
 
 
-@pytest.mark.skipif(
-    sys.version_info < (3, 5), reason="test is flakey with py2, ignore for now"
-)
```

Comment:
We should rather remove `py2` and `py35`from `tox` environments, we don't support it anymore.